### DISCUSSION TITLE
Colors of Mofiqul/vscode.nvim look washed-out

### DISCUSSION THREAD

**Discussion by rofrol** [LazyVim/LazyVim#865] [Q&A]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.9.1

### Operating system/version

MacOS 13.4

### Describe the bug

I have washed-out colors (pastels?) when using using vscode theme.

To have previous colors I have commented out `colorscheme = "vscode"`

Then after starting neovim, I switch to vscode color scheme and colors are good.

I think this happened after upgrading to neovim 0.9.1

These are my settings

```lua
  {
    "Mofiqul/vscode.nvim",
    name = "vscode",
    config = function()
      local c = require("vscode.colors").get_colors()
      require("vscode").setup({
        -- Alternatively set style in setup
        -- style = 'light'

        -- Enable transparent background
        transparent = true,

        -- Enable italic comment
        italic_comments = false,

        -- Disable nvim-tree background color
        disable_nvimtree_bg = true,

        -- Override colors (see ./lua/vscode/colors.lua)
        color_overrides = {
          -- vscLineNumber = "#FFFFFF",
        },

        -- Override highlight groups (see ./lua/vscode/theme.lua)
        group_overrides = {
          -- this supports the same val table as vim.api.nvim_set_hl
          -- use colors from this colorscheme by requiring vscode.colors!
          Cursor = { fg = c.vscDarkBlue, bg = c.vscLightGreen, bold = true },
        },
      })
      --require("vscode").load()
    end,
  },
```

This is how it looks like when I have `colorscheme = "vscode"`

<img width="803" alt="Screenshot 2023-05-31 at 13 45 17" src="https://github.com/LazyVim/LazyVim/assets/28787/2c91af9d-5fab-4eaf-a621-980d78cc845f">

And this is how it looks like when I comment out `colorscheme = "vscode"` and set colorscheme manually to vscode after starting nvim:

<img width="794" alt="Screenshot 2023-05-31 at 13 46 38" src="https://github.com/LazyVim/LazyVim/assets/28787/62c2043e-634d-482a-b28c-6db27659894d">

### Steps To Reproduce

1. Install vscode theme
2. Set it as default.
3. Colors are washed-out.

### Expected Behavior

Colors are looking like it should.

### Repro

```Lua
-- DO NOT change the paths and don't remove the colorscheme
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", lazypath, })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  "folke/tokyonight.nvim",
  "folke/LazyVim",
  -- add any other plugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

vim.cmd.colorscheme("tokyonight")
-- add anything else here
```

**Comment by folke** [comment]
Are you sure you're loading the correct colorscheme?
You make the bg transparent. Maybe the dark vscode theme is loading instead. Disable that transparent and see what loads.

Be aware that Neovim's bg detection is async, so it's very likley that it hasn't yet detected the bg when the colorscheme loads.

**Comment by rofrol** [comment]
Ok, somehow now style is needed:

`style = "light",`

Thanks
